libs/hmpb/src/vx_default_ballot_template.tsx

```diff
@@ -378,7 +535,7 @@ function CandidateContest({
                 top: 0.8,
                 left: -0.9,
                 bottom: 0.2,
-                right: 8.5,
+                right: 8.7,
```

Comment:
I lowered contest option padding from `0.75rem` to `0.5rem` so this accounts for that by expanding the write-in detection area